FizzBuzz/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Security.Cryptography;
+
+namespace FizzBuzz
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Run();
+        }
+
+        static void Run()
+        {
+            bool changed = false;
```

Comment:
Interesting solution! Do you think you could refactor it to not need the `changed` variable? 🙂